File: KyGreenPet/app/src/main/java/cc/bitky/kygreenpet/bean/KyCommand.java
package cc.bitky.kygreenpet.bean;

import cn.bmob.v3.BmobObject;

public class KyCommand extends BmobObject {

  public Boolean isOpenedLed;

  public Boolean getOpenedLed() {
    return isOpenedLed;
  }

  public void setOpenedLed(Boolean openedLed) {
    isOpenedLed = openedLed;
  }
}

File: KyGreenPetWPF/KyGreenPet/bean/KyCommand.cs
using System;
using cn.bmob.io;

namespace KyGreenPet.bean
{
    public class KyCommand : BmobTable
    {
        public BmobBoolean isOpenedLed { set; get; } = true;

        //读字段信息
        public override void readFields(BmobInput input)
        {
            base.readFields(input);
            isOpenedLed = input.getBoolean("isOpenedLed");
        }

        //写字段信息
        public override void write(BmobOutput output, bool all)
        {
            base.write(output, all);
            output.Put("isOpenedLed", isOpenedLed);
        }
    }
}